### PR TITLE
[Test] Clarification to Build Documentation

### PR DESCRIPTION
**Pull request by AlexZakIntel**
This change is [Reviewable](https://reviewable.io/reviews/isl-org/open3d/4284)

### CODE REVIEW

File: docs/builddocs.rst [hunk 1]
```diff
@@ -40,6 +40,7 @@ First, install a TeX distribution such as `MacTeX <http://www.tug.org/mactex/>`_
 
 .. code-block:: bash
 
+    # Navigate to the Open3D directory you have cloned
```
Comment (ssheorey) [inline docs/builddocs.rst:43]: Can you add this before the code block (in the regular text)?

**Review by ssheorey — CHANGES_REQUESTED** [review]

**Comment by ssheorey** [comment]
1_Overview

**Comment by AlexZakIntel** [comment]
1_Overview
1_Cube
2_Sphere
4 PCD
9_LineSet
9a_line_width
9b_more_complex_line_set
5 Multiple_obj
5a Sep_Multiple_obj
5b Multiple_obj_UI
6d_Window_Title_dimensions
6 Geoms
6a Geoms_w_compute_normals
6b_Geoms_w_UI
6c_Geoms_named_w_UI
7_show_skybox
7b_bg_color
8_point_size
10_monkey
10a_monkey_material
10b_mat_shader
11_monkey_model
12_monkey_ls
13_scale_ls_w_object
14a_cloning_gltf
15_helmet
15a_read_triangle_mesh

**Comment by AlexZakIntel** [comment]
16_index_0
16a_hose_low
16b_RubberWood_low
16c_GlassPlastic_low
16d_MetalParts_low
16e_LeatherParts_low
16f_Lenses_low
17a_Hose_low
17b_RubberWood_low
17c_GlassPlastic_low
17d_MetalParts_low
17e_LeatherParts_low
17f_Lenses_low

**Comment by AlexZakIntel** [comment]
13_scale_ls_w_object_a

**Comment by AlexZakIntel** [comment]
13_scale_ls_w_object_a
14a_cloning_gltf_a


2b1  Lit Sphere
2b2  Colored Lit Sphere
2b3  Drawing a Sphere With Materials

**Comment by AlexZakIntel** [comment]
compute_triangle_normals

**Comment by AlexZakIntel** [comment]
1a_cube_compute_vertex_normals

**Comment by AlexZakIntel** [comment]
2a_Sphere_compute_vertex_normals

**Comment by AlexZakIntel** [comment]
1a_cube_compute_triangle_normals
2a_Sphere_compute_triangle_normals_faceted_sphere

**Comment by AlexZakIntel** [comment]
3_metallic_sphere

3a_glossy_non_metallic_sphere

**Comment by AlexZakIntel** [comment]
3b_textured_sphere

3c_aabb_lineset

3d_TriangleMesh_LineSet_sphere

3d_TriangleMesh_LineSet_line_width_sphere

3d_TriangleMesh_LineSet_line_width1_sphere

3e_1_ui_win_title_dimensions

3e_2_ui_named_objects

3e_3_named_geometry_objects

**Comment by AlexZakIntel** [comment]
3f_1_show_skybox_and_bg_color

3f_2_bg_color_green

**Comment by AlexZakIntel** [comment]
3f_1_show_skybox_and_bg_color

3f_2_bg_color_green

**Comment by AlexZakIntel** [comment]
3c_aabb_lineset_line_width_50

3c_aabb_lineset_line_width_1

**Comment by AlexZakIntel** [comment]
3c_aabb_lineset_line_width_50

3c_aabb_lineset_line_width_1

**Comment by AlexZakIntel** [comment]
3c_aabb_lineset_line_width_50

3c_aabb_lineset_line_width_1

**Comment by AlexZakIntel** [comment]
1_Overview_added_math

**Comment by AlexZakIntel** [comment]
3e_2_ui_named_objects

3e_3_named_geometry_objects

3f_1_show_skybox_and_bg_color

3f_2_bg_color_green

**Comment by AlexZakIntel** [comment]
18_read_triangle_mesh

**Comment by AlexZakIntel** [comment]
dataset_splits

**Comment by AlexZakIntel** [comment]
dataset_coordinates

**Comment by AlexZakIntel** [comment]
dataset_splits

**Comment by AlexZakIntel** [comment]
3f_2_bg_color_green
2b2  Colored Lit Sphere

**Comment by AlexZakIntel** [comment]
dataset_structure

**Comment by AlexZakIntel** [comment]
dataset_coordinates

**Comment by AlexZakIntel** [comment]
1 cube_compute_vertex_normals

**Comment by AlexZakIntel** [comment]
1 cube_compute_vertex_normals

**Comment by AlexZakIntel** [comment]
3 UI_title_win_dimensions

**Comment by AlexZakIntel** [comment]
3 UI_title_win_dimensions

**Comment by AlexZakIntel** [comment]
4 Named_geometry_objects

5 Raw_mode

**Pull request closed without merging**